blocks/api/registration.js

```diff
@@ -73,6 +80,24 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
+	if ( ! ( 'category' in settings ) ) {
+		console.error(
+			'The block "' + name + '" must have a category.'
+		);
+		return;
+	}
+	if ( 'category' in settings && typeof settings.category !== 'string' ) {
+		console.error(
+			'The category slug for block "' + name + '" must be a string.'
+		);
+		return;
+	}
+	if ( 'category' in settings && findIndex( categories, ( result ) => result.slug === settings.category ) === -1 ) {
```

Comment:
Hi Aduth, thanks for your notes.
You're right in both cases. 
I will submit a new PR applying the suggestions.

Cheers,
Antonio